x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/aggregation/X-ArrayState.java.st

```diff
@@ -20,6 +20,9 @@ $endif$
 $if(double)$
 import org.elasticsearch.compute.data.IntVector;
 $endif$
+$if(float)$
```

Comment:
nit: how about merging it with the others on top?
```
$if(double|float)$
$endif$
```

Reply:
Done - [d9c2f55](https://github.com/elastic/elasticsearch/pull/109781/commits/d9c2f556218bd63342df2e8cac322d3b38b67772)